src/cmd/banner.go

```diff
@@ -0,0 +1,37 @@
+package cmd
```

Comment:
I would think that this would belong in the run package, since I don't think this is too be shared with other commands.